blockchain/chain.go

```diff
@@ -123,6 +123,18 @@ type BlockByHeightIndexEntry struct {
 	Blocks []common.Hash
 }
 
+func (bbhie *BlockByHeightIndexEntry) JsonString() string {
```

Comment:
In general, Golang might have lib to convert object to JSON string? 

e.g. https://stackoverflow.com/questions/8270816/converting-go-struct-to-json
json.Marshal()

Reply:
ack, wish i would've known it earlier!